api/api/controllers/cluster_controller.py

```diff
@@ -322,11 +268,7 @@ def get_stats_node(node_id, pretty=False, wait_for_complete=False, date=None):
     :param date: Selects the date for getting the statistical information. Format YYYY-MM-DD.
     """
     if date:
-        try:
-            year, month, day = date.split('-')
-        except ValueError:
-            raise WazuhError(1412)
-
+        year, month, day = date.split('-')

```

Comment:
We should better transform the date to a real `date` object and then extract year, month and day from it via properties.

Reply:
Done